MantidQt/CustomInterfaces/src/Muon/MuonAnalysisResultTableTab.cpp

```diff
@@ -25,7 +25,9 @@
 #include <QMessageBox>
 
 #include <algorithm>
-
+namespace {
```

Comment:
Sorry to be picky but can we have a new line between the `#include` & `namespace` and also between the closing `}` and the start of `namespace MantidQt`.

Also, please use uppercase letters for static constants - I would suggest renaming `running` to `RUNNING_LOG_NAME`.